src/main/java/nextstep/courses/domain/FreeSession.java

```diff
@@ -0,0 +1,29 @@
+package nextstep.courses.domain;
+
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class FreeSession extends Session {
+    public FreeSession(Long id, String title, LocalDate startDate, LocalDate endDate, CoverImage coverImage, SessionStatus status) {
+        super(id, title, startDate, endDate, coverImage, status);
+    }
+
+    public FreeSession(Long id, String title, LocalDate startDate, LocalDate endDate, CoverImage coverImage, SessionStatus status, List<NsUser> userList) {
+        super(id, title, startDate, endDate, coverImage, status, userList);
+    }
+
+    @Override
+    public void join(NsUser user) {
+        if(this.status != SessionStatus.RECRUITING){
```

Comment:
부분적으로 공백, 개행 등의 컨벤션이 안 맞는 경우가 종종있네요
IDE의 기능을 활용해 저장할 때 포맷팅하는 기능등을 활용하시면 좋을 것 같아요 

Enum 타입도 하나의 객체이니 메시지를 보내면 어떨까요?
```suggestion
        if (status.isRecruiting()) {
```